packages/style/package.json

```diff
@@ -13,31 +13,25 @@
     "url": "git+ssh://git@github.com/Ferlab-Ste-Justine/ferlab-ui.git"
   },
   "scripts": {
-    "storybook": "npm run storybook",
     "compile:colors": "./bin/style-script.js colors",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Ferlab - Centre de recherche du CHU Ste-Justine",
   "license": "SEE LICENSE IN LICENSE",
   "devDependencies": {
-    "@types/antd": "^1.0.0",
-    "@types/react": "^16.9.51",
-    "@types/react-router-dom": "^5.1.6",
-    "npm-run-all": "^4.1.5",
-    "sass": "^1.26.12",
-    "typescript": "^4.0.3",
-    "webpack-cli": "^3.3.12",
-    "command-line-args": "^5.1.1",
-    "cross-spawn": "^7.0.3",
-    "less": "^4.1.0",
-    "less2sass": "^1.0.3"
+    "antd": "^4.12.3"
   },
   "peerDependencies": {
     "react": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "antd": "^4.11.1"
+    "antd": "^4.12.3"
+  },
+  "dependencies": {
+    "command-line-args": "^5.1.1",
+    "cross-spawn": "^7.0.3",
+    "less": "^4.1.0",
+    "less2sass": "^1.0.3"
```

Comment:
dev?

Reply:
If we want to use the script wihout npm link to compile the less to sass files those needs to be installed on the real projects.
We could move the script to a script package and installed it in the dev section of the main project though to remove it from prod project...

Reply:
The trouble with that, it will install the depencies on all project s that uses it, without any need for them